test/unit/eigensolver/mc/CMakeLists.txt

```diff
@@ -8,6 +8,13 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
+DLAF_addTest(test_backtransformation
+  SOURCES test_backtransformation.cpp
+  LIBRARIES dlaf.eigensolver dlaf.core
+  USE_MAIN MPIHPX
+  MPIRANKS 1
```

Comment:
So far it uses one rank only to deal with `computeTFactor` requirement (as you noticed in another comment). But as soon as the distributed version of the algorithm will be implemented it will be set to 6 ranks.